src/components/loader/loader.scss

```diff
@@ -13,14 +13,15 @@ $loader-scale: 54;
 $loader-circumference: ($loader-scale - (2 * $stroke-width)) * 3.14159;
 
 :host {
-  @apply relative mx-auto hidden items-center justify-center py-16 opacity-100;
+  @apply relative mx-auto hidden items-center justify-center opacity-100;
   min-block-size: var(--calcite-loader-size);
   font-size: var(--calcite-loader-font-size);
   stroke: var(--calcite-ui-brand);
   stroke-width: $stroke-width;
   fill: none;
   transform: scale(1, 1);
   animation: loader-color-shift 6s alternate-reverse infinite linear;
+  padding-block: var(--calcite-loader-padding, 4rem);
```

Comment:
why the 4rem here?

Reply:
replacing py-16, spacing.16 would be 4rem in tailwind.

Reply:
Can you reference the spacing instead of 4rem? `theme("spacing.16")`?

Reply:
The problem with 4rem is that if we change the spacing down the road it won't get updated here.

Reply:
agreed. better to refer tailwind utils.